### PR TITLE
Properties changed event

### PR DESCRIPTION
**Pull request by clayrisser**
I added an event that fires after the properties have been updated.  It is extremely inefficient and difficult for me to use this element without an event that fires after the properties have been updated.  Without this event, I have to create an observer that listens for the properties to change.  This is ok except for when I have multiple properties in a route.  If I try and listen for the properties to change with multiple properties, the observers fire once for each property, and the first time it fires, the second property hasn't been set yet, so I have to program my software to ignore the first time the event fires.  Additionally, if I have the same variable used for different routes, the observers will not fire when the route changes if the variables don't change.  For example, if http://localhost/foo/bar changes to http://foo, and 'foo' is bound to the same variable in both routes, my observer will not fire.

In summary, it is extremely difficult and almost impossible to use excess-router in certain advanced cases without an event firing after all the properties have been set.  I would appreciate if you would accept this pull request so I don't have to create my own branch of excess-router and worry about keeping it updated.  If you have a better way of doing this, please let me know.

### CODE REVIEW

**Comment by atotic** [comment]
Why don't you use multiple property observer:
https://www.polymer-project.org/1.0/docs/devguide/properties.html#multi-property-observers

PS: Polymer project has just released carbon-router, I am hoping that it'll become a new standard.

**Comment by clayrisser** [comment]
I guess I wasn't familiar with the multiple property observer.  Thanks so much.  That should work.  I still think it is odd that this element doesn't fire an event after the properties are set, but I guess a multiple property observer will work.

**Comment by clayrisser** [comment]
I'm kind of excited about a polymer supported router.

**Comment by clayrisser** [comment]
Supposedly the carbon router is deprecated.  I'm going to use excess-router for now.

**Comment by atotic** [comment]
https://github.com/PolymerElements/carbon-route

**Comment by clayrisser** [comment]
Ah, thanks
